tests/tvos/view-with-example/tvos-view-with-example.test.js

```diff
@@ -1,84 +1,85 @@
-const mockCallSnapshot = []
+const mockCallSnapshot = [];
```

Comment:
Here is an example where `lint:fix` did **not** clean up after the quick `npx prettier --write tests` that was done resolve some indenting issues.

Reply:
... but `lint:fix` **does** clean this up on my Windows machine.